README.md

```diff
@@ -1,6 +1,9 @@
 # [Shopify](https://www.shopify.com) JavaScript Buy SDK
 [![Circle CI](https://circleci.com/gh/Shopify/js-buy-sdk.png?circle-token=3be0ebe6fbb4841442b86678696947bd4b5456d7)](https://circleci.com/gh/Shopify/js-buy-sdk)
 
+**Note**: For help with migrating from v0.7 of JS Buy SDK to v1 check out the
+[Migration Guide](https://github.com/Shopify/js-buy-sdk/blob/v1.0alpha/docs/MIGRATION_GUIDE.md).
+
```

Comment:
It's set up to link to the v1.0alpha branch, so it should work after this gets merged.